src/main/java/ch/njol/skript/expressions/ExprSkullOwner.java

```diff
@@ -1,40 +1,48 @@
 package ch.njol.skript.expressions;
 
+import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.bukkitutil.ItemUtils;
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.coll.CollectionUtils;
 import org.bukkit.Bukkit;
 import org.bukkit.OfflinePlayer;
 import org.bukkit.block.Block;
-import org.bukkit.block.BlockState;
 import org.bukkit.block.Skull;
 import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
 import org.jetbrains.annotations.Nullable;
 
 @Name("Skull Owner")
 @Description("The skull owner of a player skull.")
 @Examples({
 	"set {_owner} to the skull owner of event-block",
-	"set skull owner of {_block} to \"Njol\" parsed as offlineplayer"
+	"set skull owner of {_block} to \"Njol\" parsed as offlineplayer",
+	"set skull owner of player's tool to \"Njol\" parsed as offlineplayer"
```

Comment:
Suggestion for a different value such as `{_player}` since the one above already showcased one of the possibilities